core/templates/dev/head/filters.js

```diff
@@ -240,6 +240,7 @@ oppia.filter('parameterizeRuleDescription', [
           replacementText = inputs[varName];
         }
 
+        replacementText = replacementText.replace('$', '$$$$');
```

Comment:
Please add a comment in code; what's going on here is not obvious. Also see if you can find a way to add Karma tests.

Thanks!

Reply:
Did that. Added a karma test too